keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -513,6 +513,17 @@ kpxc.prepareCredentials = async function() {
 
     kpxc.initLoginPopup();
     kpxc.initAutocomplete();
+
+    if (kpxc.settings.autoFillRelevantCredential) {
+        const pageUuid = await sendMessage('page_get_login_id');
+        if (pageUuid) {
+            const relevantCredential = kpxc.credentials.find(c => c.uuid === pageUuid);
+            const combination = kpxc.combinations?.at(-1);
```

Comment:
Does this return the last combination pair discovered on the page?

Reply:
I think this is here because if there are two combinations, the latter is the one with the password field. Still, it wouldn't hurt to actually check that the combination has that field.